design/sketch.webidl

```diff
@@ -387,23 +388,23 @@ enum GPUInputStepMode {
 };
 
 dictionary GPUVertexAttributeDescriptor {
-    u32 shaderLocation;
-    u32 inputSlot;
     u32 offset;
     GPUVertexFormat format;
+    GPUShaderAttributeIndex attributeIndex;
```

Comment:
Presumably this has the same meaning as the previously-existing "u32 shaderLocation"?

Reply:
Yes.